lua/neo-tree/ui/renderer.lua

```diff
@@ -834,6 +834,7 @@ local set_buffer_mappings = function(state)
         end
         if type(func) == "string" then
           resolved_mappings[cmd] = { text = func }
+          map_options.desc = func
```

Comment:
I think this should preserve the provided `desc` in case it was set. Someone might want to have a description that's different from the function name.
```suggestion
          map_options.desc = map_options.desc or func
```